OpenMEEG/include/GeometryIOs/GeomFile.h

```diff
@@ -298,7 +298,7 @@ namespace OpenMEEG::GeometryIOs {
 
         //  Create the list of interfaces.
 
-        std::vector<const Mesh*> meshes;
+        std::vector<const Mesh*> these_meshes;
```

Comment:
There are a lot of these "shadowing a class variable", hence the many renames

Reply:
In this case, there is a better fix. The class member should not exist and I think there is a potential bug hidden (but in an obscure corner case where we provide both a mesh file and a mesh section. I will correct that in the branch.

Reply:
Sounds good!